recipes/apstools/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "apstools" %}
+{% set version = "1.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/apstools-{{ version }}.tar.gz
+  sha256: 459747b504df5def3564bf6e4d990e8b7eff42910dbeeb1dd73f9a76c9960649
+
+build:
+  entry_points:
+    - bluesky_snapshot = apstools.snapshot.application:snapshot_cli
+    - bluesky_snapshot_viewer = apstools.snapshot.application:snapshot_gui
+    - spec2ophyd = apstools.migration.spec2ophyd:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py>=36 and py<310]
```

Comment:
Skipping >=3.6 and <3.10 is the same as just <3.10 b/c we do not build <3.5. No need for the complex skip.